draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -466,14 +455,13 @@ grammar:
 ~~~ abnf
 set-cookie-header = "Set-Cookie:" SP set-cookie-string
 set-cookie-string = cookie-pair *( ";" SP cookie-av )
-cookie-pair       = cookie-name "=" cookie-value
-cookie-name       = token
+cookie-pair       = cookie-name *[WSP] "=" *[WSP] cookie-value
+cookie-name       = 1*cookie-octet
```

Comment:
This is not enough.
What about these:

1. `Set-Cookie: a=b ;   sameSite=strict`
2. `SetCookie: a=b ; sameSite = strict`
3. `SameSite:  a=b; sameSite=lax ; path = /`

Reply:
*[WSP]

is the same as

    *WSP

Reply:
Thanks to you both. I've replaced WSP with OWS (I'd mistakenly assumed RFC7230's OWS contained a wider character set than WSP), and added more flexibility around delimiters, in line with the WPT suite's tests.

WDYT?